src/ZPublisher/BeforeTraverse.py

```diff
@@ -86,7 +86,11 @@ class MultiHook(object):
     MultiHook calls the named hook from the class of the container, then
     the prior hook, then all the hooks in its list.
     """
-    def __init__(self, hookname, prior, defined_in_class):
+    def __init__(self, hookname=None, prior=None, defined_in_class=None):
```

Comment:
I think an explicit string would be a better default for `hookname` (maybe `'<undefined hookname>'`?) and `False` would be a better default for `defined_in_class`.